load_hours.py

```diff
@@ -10,14 +10,26 @@
 harvest_client = HarvestClient()
 jira_client = JiraClient()
 
-date_from = date.today().isoformat()
-date_to = date.today().isoformat()
+# Track the last day it ran
+with open('.aux', 'r') as f:
+    last_day = f.read()
+    if not last_day:
+        last_day = date.today()
+
+if isinstance(last_day, str):
+    last_day = datetime.strptime(last_day, '%d/%m/%Y').date()
+
+date_from = last_day
+date_to = date.today() - timedelta(days=1)
+
+with open('.aux', 'w+') as f:
```

Comment:
I think we should move this write logic to after the `load_hours` main logic is done. Because if some error occurs during the load, it will consider that all hours of the date were loaded.